docs/luigi-core-api.md

```diff
@@ -587,6 +587,26 @@ Luigi.navigation().navigate('users/groups/stakeholders')
 Luigi.navigation().navigate('/settings', null, true) // preserve view
 ```
 
+#### navigateToIntent
+
+Offers an alternative way of navigating with intents. This involves specifying a semanticSlug and an object containing parameters.
+
+##### Parameters
+
+-   `semanticSlug` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** concatenation of semantic object and action connected with a dash (-)
+-   `params` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** an object representing all the parameters passed (optional, default '{}')
+
+##### Examples
+
+```javascript
+Luigi.navigation().navigateToIntent('Sales-settings')
+Luigi.navigation().navigateToIntent('Sales-settings', {project: 'pr1'})
+```
+
+**Meta**
+
+-   **since**: NEXTRELEASE
```

Comment:
I think `NEXTRELEASE` should be in the core-api/navigation.js and then be created automatically here